Makefile

```diff
@@ -295,16 +337,20 @@ test-internal:
 	@grep '^FAIL' test.log || true
 	@if [ "$$(cat exit-code)" == "0" ] ; then echo "PASS" ; exit 0 ; else exit 1 ; fi
 
-test-all: other-consul dev-build lint $(foreach mod,$(GO_MODULES),test-module/$(mod))
+.PHONY: test-all
+test-all: other-consul dev-build lint $(foreach mod,$(GO_MODULES),test-module/$(mod)) ## Test all
 
+.PHONY: test-module/%
 test-module/%:
 	@echo "--> Running go test ($*)"
 	cd $* && go test $(GOTEST_FLAGS) -tags '$(GOTAGS)' ./...
 
-test-race:
+.PHONY: test-race
+test-race: ## Test race
 	$(MAKE) GOTEST_FLAGS=-race
 
-test-docker: linux go-build-image
+.PHONY: test-docker
+test-docker: linux go-build-image ## Running tests within a docker container
```

Comment:
I wonder if anyone still uses this. I think we added it to be able to isolate and run tests with less resources than taking over our whole machine. Since then we have reduced test parallelism. Might be worth a quick poll of developers to see if anything thinks we should keep this.

Reply:
Did ask this in the channel and no one responded this so I guess no cares . Removed this command and hopefully no one will be grumpy 😂